.babelrc

```diff
@@ -1,7 +1,13 @@
 {
   // Javascript settings.
   "presets": [
-    "es2015",
+    ["env", {
+      "targets": {
+        "uglify": true,
+        "browsers": ["last 2 versions", "ie 11", "not ie 10"]
```

Comment:
For whatever it's worth, [this](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/4eaa95c7f30f8f4b78bdb8ba2ebe0d7d27dd7763/Decisons/Browser%20Support.md) is what is currently documented as our browser support policy, but I believe that needs updating. 

[This](https://github.com/department-of-veterans-affairs/vets-website/pull/3716/files) PR is what implemented telling users with out of date browsers to update if you want to see the code. 

And [this](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/358) issue is when the decision to make the minimum supported version IE11 was made. 

Phew, that's entirely disorganized and should be combined somewhere (likely in the first link?)